pydra/engine/audit.py

```diff
@@ -214,15 +214,15 @@ def audit_task(self, task):
         start_message = {
             "@id": self.aid,
             "@type": "task",
-            "Label": label,
+            "Activity_Label": label,
```

Comment:
@djarecka You will see that I have change "Label" to "Acitivty_Label" and "Entity_Label", respecitvely. I'm not sure if there exists a means of parsing JSON with duplicate key entries (i.e., two "Labels"). This is my proposed workaround in the meantime.

Reply:
Note that the duplicate entry is what is in the BEP document, so it will need to be changed there as well.

Reply:
what do you mean by duplicate entry?